pkg/probe/probes.go

```diff
@@ -188,8 +195,13 @@ func runPing(_ client.Client) (bool, error) {
 	return true, nil
 }
 
-func ping(target string) (string, error) {
-	cmd := exec.Command("ping", "-c", "1", target)
+func ping(target Route) (string, error) {
+	// If next hop is IPv6 link-local, we need to append an interface otherwise it is
+	// not clear which interface should be used for communication (e.g. ping test).
+	// As this syntax works always, we simply append it always.
+	dest := target.nextHop.String() + target.iface
```

Comment:
Did we lose the % separator here or is it included in one of those values?

Reply:
Of course we did. I was wondering why all the tests failed and I was just starting to unfold their cryptic failure messages